package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sanctuary-def",
+  "version": "0.0.0",
+  "description": "Define strict, curried functions",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/plaid/sanctuary-def.git"
+  },
+  "dependencies": {
+    "ramda": "0.18.x"
```

Comment:
I agree! Ramda was helpful in getting this project up and running, but I'm now keen to remove the dependency. I plan to spend time this weekend sanding a few rough edges before merging this pull request and publishing the package. I'll remove the Ramda dependency while I'm at it. :)